test/built-ins/TypedArrays/from/mapfn-is-not-callable.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-%typedarray%.from
+description: Throw a TypeError exception is mapfn is not callable
+info: >
+  22.2.2.1 %TypedArray%.from ( source [ , mapfn [ , thisArg ] ] )
+
+  ...
+  3. If mapfn was supplied and mapfn is not undefined, then
+    a. If IsCallable(mapfn) is false, throw a TypeError exception.
+  ...
+includes: [testTypedArray.js]
+features: [Symbol.iterator]
+---*/
+
+var arrayLike = {};
+Object.defineProperty(arrayLike, Symbol.iterator, {
```

Comment:
Ah, now I understand. That's a great detail to check! But as written, it *is* a
little implicit. A comment would help, but an explicit assertion might be even
better. Like a comment, ensuring that a counter was not incremented would make
the test easier to read, but it would also make the failure more clear.

```js
assert.sameValue(callCount, 0, 'The @@iterator method should not be invoked.');
```